### ISSUE TITLE
Memory crash on load

### ISSUE THREAD

**Issue by Augustin37** [Zrips/Jobs#35]
Hey, I have a memory error when I load the latest revision of the plugin: Here is the java error: 
```java
[01:11:25] [Jobs-BufferedPaymentThread/INFO]: [0;33;1m[Jobs] Started buffered payment thread.[m
[01:11:55] [Server thread/ERROR]: Error occurred while enabling Jobs v3.5.6 (Is it up to date?)
java.lang.OutOfMemoryError: GC overhead limit exceeded
	at net.minecraft.server.v1_9_R2.NBTBase.createTag(SourceFile:89) ~[patched_1.9.4.jar:git-Paper-772]
	at net.minecraft.server.v1_9_R2.NBTTagList.load(NBTTagList.java:51) ~[patched_1.9.4.jar:git-Paper-772]
	at net.minecraft.server.v1_9_R2.NBTTagCompound.a(NBTTagCompound.java:386) ~[patched_1.9.4.jar:git-Paper-772]
	at net.minecraft.server.v1_9_R2.NBTTagCompound.load(NBTTagCompound.java:46) ~[patched_1.9.4.jar:git-Paper-772]
	at net.minecraft.server.v1_9_R2.NBTTagCompound.a(NBTTagCompound.java:386) ~[patched_1.9.4.jar:git-Paper-772]
	at net.minecraft.server.v1_9_R2.NBTTagCompound.load(NBTTagCompound.java:46) ~[patched_1.9.4.jar:git-Paper-772]
	at net.minecraft.server.v1_9_R2.NBTTagCompound.a(NBTTagCompound.java:386) ~[patched_1.9.4.jar:git-Paper-772]
	at net.minecraft.server.v1_9_R2.NBTTagCompound.load(NBTTagCompound.java:46) ~[patched_1.9.4.jar:git-Paper-772]
	at net.minecraft.server.v1_9_R2.NBTTagList.load(NBTTagList.java:53) ~[patched_1.9.4.jar:git-Paper-772]
	at net.minecraft.server.v1_9_R2.NBTTagCompound.a(NBTTagCompound.java:386) ~[patched_1.9.4.jar:git-Paper-772]
	at net.minecraft.server.v1_9_R2.NBTTagCompound.load(NBTTagCompound.java:46) ~[patched_1.9.4.jar:git-Paper-772]
	at net.minecraft.server.v1_9_R2.NBTTagCompound.a(NBTTagCompound.java:386) ~[patched_1.9.4.jar:git-Paper-772]
	at net.minecraft.server.v1_9_R2.NBTTagCompound.load(NBTTagCompound.java:46) ~[patched_1.9.4.jar:git-Paper-772]
	at net.minecraft.server.v1_9_R2.NBTTagCompound.a(NBTTagCompound.java:386) ~[patched_1.9.4.jar:git-Paper-772]
	at net.minecraft.server.v1_9_R2.NBTTagCompound.load(NBTTagCompound.java:46) ~[patched_1.9.4.jar:git-Paper-772]
	at net.minecraft.server.v1_9_R2.NBTTagList.load(NBTTagList.java:53) ~[patched_1.9.4.jar:git-Paper-772]
	at net.minecraft.server.v1_9_R2.NBTTagCompound.a(NBTTagCompound.java:386) ~[patched_1.9.4.jar:git-Paper-772]
	at net.minecraft.server.v1_9_R2.NBTTagCompound.load(NBTTagCompound.java:46) ~[patched_1.9.4.jar:git-Paper-772]
	at net.minecraft.server.v1_9_R2.NBTTagCompound.a(NBTTagCompound.java:386) ~[patched_1.9.4.jar:git-Paper-772]
	at net.minecraft.server.v1_9_R2.NBTTagCompound.load(NBTTagCompound.java:46) ~[patched_1.9.4.jar:git-Paper-772]
	at net.minecraft.server.v1_9_R2.NBTTagCompound.a(NBTTagCompound.java:386) ~[patched_1.9.4.jar:git-Paper-772]
	at net.minecraft.server.v1_9_R2.NBTTagCompound.load(NBTTagCompound.java:46) ~[patched_1.9.4.jar:git-Paper-772]
	at net.minecraft.server.v1_9_R2.NBTTagList.load(NBTTagList.java:53) ~[patched_1.9.4.jar:git-Paper-772]
	at net.minecraft.server.v1_9_R2.NBTTagCompound.a(NBTTagCompound.java:386) ~[patched_1.9.4.jar:git-Paper-772]
	at net.minecraft.server.v1_9_R2.NBTTagCompound.load(NBTTagCompound.java:46) ~[patched_1.9.4.jar:git-Paper-772]
	at net.minecraft.server.v1_9_R2.NBTCompressedStreamTools.a(NBTCompressedStreamTools.java:84) ~[patched_1.9.4.jar:git-Paper-772]
	at net.minecraft.server.v1_9_R2.NBTCompressedStreamTools.a(NBTCompressedStreamTools.java:53) ~[patched_1.9.4.jar:git-Paper-772]
	at net.minecraft.server.v1_9_R2.NBTCompressedStreamTools.a(NBTCompressedStreamTools.java:23) ~[patched_1.9.4.jar:git-Paper-772]
	at net.minecraft.server.v1_9_R2.WorldNBTStorage.getPlayerData(WorldNBTStorage.java:215) ~[patched_1.9.4.jar:git-Paper-772]
	at org.bukkit.craftbukkit.v1_9_R2.CraftOfflinePlayer.getData(CraftOfflinePlayer.java:172) ~[patched_1.9.4.jar:git-Paper-772]
	at org.bukkit.craftbukkit.v1_9_R2.CraftOfflinePlayer.getBukkitData(CraftOfflinePlayer.java:176) ~[patched_1.9.4.jar:git-Paper-772]
	at org.bukkit.craftbukkit.v1_9_R2.CraftOfflinePlayer.getLastPlayed(CraftOfflinePlayer.java:214) ~[patched_1.9.4.jar:git-Paper-772]
[01:11:55] [Server thread/INFO]: [Jobs] Disabling Jobs v3.5.6
```
I think the plugin is looking for informations on offline players. My server is very old, so it has a lot of players to check for, maybe this cause the crash?

Thank you for your help :)

**Comment by Augustin37** [comment]
And it do also the same thing when "UseLocalOfflinePlayersData" is set to false in the config

**Comment by Augustin37** [comment]
Can you help me please? I can't update my server on 1.10 because of this :/

**Comment by Augustin37** [comment]
This issue is fixed with the version 3.5.7 (Beta) Thank you :)
